velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -3836,3 +3836,18 @@ TEST_F(DateTimeFunctionsTest, fromUnixtimeDouble) {
   });
   assertEqualVectors(expected, actual);
 }
+
+TEST_F(DateTimeFunctionsTest, toISO8601TestDate) {
```

Comment:
toISO8601TestDate -> toIso8601

Test and Date suffixes are redundant.

Reply:
Removed "Test" suffix, but I wanted to keep each signature in its own test class, as the others are fairly lengthier and can get cluttered quick.